ethereum/contracts/@openzeppelin/contracts/utils/Context.sol

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: MIT
 // OpenZeppelin Contracts v4.4.1 (utils/Context.sol)
 
-pragma solidity ^0.8.19;
+pragma solidity ^0.8.0;
```

Comment:
Starting with this PR we don't modify or format the openzepplin contracts, making diffs with the official releases clean.